src/command.test.js

```diff
@@ -7,24 +7,36 @@ const { Command } = require('./command')
 describe('Command', () => {
   beforeEach(() => { clearAllMocks() })
 
-  describe('empty command', () => {
-    class TestCommand extends Command {
-      // TODO: This will be static later, will be addressed in other PR.
-      schema =
-        {
+  describe('with empty schema', () => {
+    describe('explicit version', () => {
+      describe('static run', () => {
+        class TestCommand extends Command {
+          schema = {}
 
+          execute () {
+            return 'We made it!'
+          }
         }
+        it.only('executes successfully', async () => {
```

Comment:
I removed this `.only` in the following PR. Tests are green on that one, no need to worry.